tests/fsharp/optimize/stats/stats.txt

```diff
@@ -452,3 +452,4 @@ Tue 07/14/2009,  9:26:43.59, Microsoft.FSharp-TypeFunc, 0, Microsoft.FSharp-clas
 20/01/2010, 19:59:48.26, Microsoft.FSharp-TypeFunc, 0, Microsoft.FSharp-classes, 1368,  Microsoft.FSharp-methods, 5716, ,  Microsoft.FSharp-fields, 1933,  
 21/04/2010, 21:42:44.95, Microsoft.FSharp-TypeFunc, 0, Microsoft.FSharp-classes, 1376,  Microsoft.FSharp-methods, 5730, ,  Microsoft.FSharp-fields, 1945,  
 Sun 11/09/2014, 17:38:09.23, Microsoft.FSharp-TypeFunc, 0, Microsoft.FSharp-classes, 2504,  Microsoft.FSharp-methods, 8641, ,  Microsoft.FSharp-fields, 3827,  
+Sat 02/07/2015,  0:35:46.24, Microsoft.FSharp-TypeFunc, 0, Microsoft.FSharp-classes, 2557,  Microsoft.FSharp-methods, 8939, ,  Microsoft.FSharp-fields, 4018,  
```

Comment:
It's an old tracking file that gets updated whenever you runs all the tests, isn't related to the PR at all. Should probably be added to .gitignore at some point, since it doesn't serve much purpose any more. As you can see, only 1 update in the past 5 years :)